src/main/java/soundchan/BotListener/UserSoundWatcher.java

```diff
@@ -0,0 +1,69 @@
+package soundchan.BotListener;
+
+import soundchan.LocalAudioManager;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.*;
+
+import static java.nio.file.StandardWatchEventKinds.ENTRY_MODIFY;
+
+public class UserSoundWatcher implements Runnable {
+
+    private LocalAudioManager localAudioManager;
+    private String userSoundFile;
+    private Path soundFileDirectory;
+    private WatchService watchService;
+    private WatchKey watchKey;
+
+    @SuppressWarnings("unchecked")
+    static <T> WatchEvent<T> cast(WatchEvent<?> event) {
+        return (WatchEvent<T>) event;
+    }
+
+    public UserSoundWatcher(LocalAudioManager audioManager, String filepath) {
```

Comment:
For simplicity, try and combine this class and Directory watcher, they are almost identical after all. You could subclass them off of the same base class (defined by you), but I'm not so sure they need to be in separate files